running: {'--uuid': '4dd5610812c15cc9b23b221141bb85b0', '-db-root': '/home/ryedida/bbo_challenge_starter_kit/output', '--opt-root': '/home/ryedida/bbo_challenge_starter_kit/example_submissions', '--data-root': None, '--db': 'run_20230427_004608', '--opt': 'smoothness', '--data': 'diabetes', '--classifier': 'MLP-adam', '--metric': 'mse', '--calls': 15, '--suggestions': 1, '--jobs-file': None, '--verbose': False, 'dry_run': False, 'rev': 'a376313', 'opt_rev': 'x.x.x'}
cmd: python smoothness/optimizer.py -c MLP-adam -d diabetes -o smoothness -u 4dd5610812c15cc9b23b221141bb85b0 -m mse -n 15 -p 1 -dir /home/ryedida/bbo_challenge_starter_kit/output -b run_20230427_004608
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

computed signature: ([{'alpha': 0.019628224813442792, 'batch_size': 182, 'beta_1': 0.9410202200271762, 'beta_2': 0.9998021557676793, 'epsilon': 1.8662266976518e-08, 'hidden_layer_sizes': 147, 'learning_rate_init': 0.0005627932047415167, 'tol': 0.03690557729213761, 'validation_fraction': 0.8846827852548593}, {'alpha': 0.0019982467392329444, 'batch_size': 200, 'beta_1': 0.919111482530466, 'beta_2': 0.9998488260436156, 'epsilon': 5.981221901152555e-07, 'hidden_layer_sizes': 61, 'learning_rate_init': 2.2310905607443014e-05, 'tol': 1.2046852412030316e-05, 'validation_fraction': 0.8117896445826539}, {'alpha': 0.46659545670218433, 'batch_size': 219, 'beta_1': 0.9889789783750891, 'beta_2': 0.9999896868093284, 'epsilon': 2.4234724484675948e-08, 'hidden_layer_sizes': 167, 'learning_rate_init': 2.972334644335654e-05, 'tol': 0.0036281404040243792, 'validation_fraction': 0.17260651658522078}, {'alpha': 4.656005689076002, 'batch_size': 135, 'beta_1': 0.870503881627747, 'beta_2': 0.9948873266941017, 'epsilon': 2.1023308743480125e-07, 'hidden_layer_sizes': 118, 'learning_rate_init': 0.0018781738757161913, 'tol': 1.1889379831773004e-05, 'validation_fraction': 0.6264327093752792}, {'alpha': 0.04705159350400542, 'batch_size': 158, 'beta_1': 0.9870884262730957, 'beta_2': 0.9999596874382349, 'epsilon': 1.1981845126013875e-08, 'hidden_layer_sizes': 116, 'learning_rate_init': 0.00617340520407431, 'tol': 1.741413418158619e-05, 'validation_fraction': 0.6754299026638921}], [29083.993548572587, 29062.423277324073, 29075.78987471409, 24626.32866834337, 9584.226858493177])
Signature errors:
                              0         1         2         3         4       max
MLP-adam_diabetes_mse  0.000007  0.000008  0.000003  0.003558  0.004952  0.004952
max                    0.000007  0.000008  0.000003  0.003558  0.004952  0.004952
starting sklearn study smoothness MLP-adam diabetes mse 15 1
with data root: None
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 49, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.494427 iter 0 next_points [{'alpha': 0.017139714591831855, 'batch_size': 220, 'beta_1': 0.6254810608622902, 'beta_2': 0.9589385495593318, 'epsilon': 7.042361369712631e-07, 'hidden_layer_sizes': 179, 'learning_rate_init': 0.022880291011895475, 'tol': 0.03250959087938017, 'validation_fraction': 0.8480370294764114}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

function_evaluation time 0.227998 value 4068.011364 suggestion {'alpha': 0.017139714591831855, 'batch_size': 220, 'beta_1': 0.6254810608622902, 'beta_2': 0.9589385495593318, 'epsilon': 7.042361369712631e-07, 'hidden_layer_sizes': 179, 'learning_rate_init': 0.022880291011895475, 'tol': 0.03250959087938017, 'validation_fraction': 0.8480370294764114}
observation time 0.000007, current best 4068.011364 at iter 0
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 49, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.329805 iter 1 next_points [{'alpha': 0.008640501236840114, 'batch_size': 240, 'beta_1': 0.7971072306458145, 'beta_2': 0.9999977760561446, 'epsilon': 7.429620364050146e-07, 'hidden_layer_sizes': 125, 'learning_rate_init': 0.0003198339427125427, 'tol': 0.012550272107105905, 'validation_fraction': 0.13600741288600246}]
function_evaluation time 0.080329 value 29067.538839 suggestion {'alpha': 0.008640501236840114, 'batch_size': 240, 'beta_1': 0.7971072306458145, 'beta_2': 0.9999977760561446, 'epsilon': 7.429620364050146e-07, 'hidden_layer_sizes': 125, 'learning_rate_init': 0.0003198339427125427, 'tol': 0.012550272107105905, 'validation_fraction': 0.13600741288600246}
observation time 0.000007, current best 4068.011364 at iter 1
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 49, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.326343 iter 2 next_points [{'alpha': 0.0001224148439486366, 'batch_size': 153, 'beta_1': 0.9873138993102634, 'beta_2': 0.999713132943914, 'epsilon': 6.059053899364376e-08, 'hidden_layer_sizes': 94, 'learning_rate_init': 1.4753834050577219e-05, 'tol': 0.0019363915166118253, 'validation_fraction': 0.39048237890030224}]
function_evaluation time 0.065801 value 29118.196881 suggestion {'alpha': 0.0001224148439486366, 'batch_size': 153, 'beta_1': 0.9873138993102634, 'beta_2': 0.999713132943914, 'epsilon': 6.059053899364376e-08, 'hidden_layer_sizes': 94, 'learning_rate_init': 1.4753834050577219e-05, 'tol': 0.0019363915166118253, 'validation_fraction': 0.39048237890030224}
observation time 0.000006, current best 4068.011364 at iter 2
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 49, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.329865 iter 3 next_points [{'alpha': 0.02598435838901931, 'batch_size': 91, 'beta_1': 0.6591220304854247, 'beta_2': 0.9999409991677616, 'epsilon': 8.066663707279113e-07, 'hidden_layer_sizes': 96, 'learning_rate_init': 0.004736363348753204, 'tol': 5.0556193479014706e-05, 'validation_fraction': 0.10586060508808898}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

function_evaluation time 0.831650 value 3204.059810 suggestion {'alpha': 0.02598435838901931, 'batch_size': 91, 'beta_1': 0.6591220304854247, 'beta_2': 0.9999409991677616, 'epsilon': 8.066663707279113e-07, 'hidden_layer_sizes': 96, 'learning_rate_init': 0.004736363348753204, 'tol': 5.0556193479014706e-05, 'validation_fraction': 0.10586060508808898}
observation time 0.000006, current best 3204.059810 at iter 3
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 49, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.327230 iter 4 next_points [{'alpha': 0.0007528772242005327, 'batch_size': 247, 'beta_1': 0.9683306178361951, 'beta_2': 0.9891857128463162, 'epsilon': 1.4584413458913327e-09, 'hidden_layer_sizes': 180, 'learning_rate_init': 1.918742535045793e-05, 'tol': 1.6236408933729003e-05, 'validation_fraction': 0.8373692699515245}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

function_evaluation time 0.408961 value 29070.133405 suggestion {'alpha': 0.0007528772242005327, 'batch_size': 247, 'beta_1': 0.9683306178361951, 'beta_2': 0.9891857128463162, 'epsilon': 1.4584413458913327e-09, 'hidden_layer_sizes': 180, 'learning_rate_init': 1.918742535045793e-05, 'tol': 1.6236408933729003e-05, 'validation_fraction': 0.8373692699515245}
observation time 0.000006, current best 3204.059810 at iter 4
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 49, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.328788 iter 5 next_points [{'alpha': 0.8857079863890182, 'batch_size': 178, 'beta_1': 0.9366384049555821, 'beta_2': 0.9999960216298197, 'epsilon': 2.8545187332449147e-09, 'hidden_layer_sizes': 60, 'learning_rate_init': 0.03695521813146302, 'tol': 0.04496160478444588, 'validation_fraction': 0.670601778392894}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

function_evaluation time 0.092053 value 17238.582877 suggestion {'alpha': 0.8857079863890182, 'batch_size': 178, 'beta_1': 0.9366384049555821, 'beta_2': 0.9999960216298197, 'epsilon': 2.8545187332449147e-09, 'hidden_layer_sizes': 60, 'learning_rate_init': 0.03695521813146302, 'tol': 0.04496160478444588, 'validation_fraction': 0.670601778392894}
observation time 0.000006, current best 3204.059810 at iter 5
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 49, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.328669 iter 6 next_points [{'alpha': 1.1645647017687961, 'batch_size': 139, 'beta_1': 0.7486998016452818, 'beta_2': 0.9902065016382664, 'epsilon': 7.344196077178596e-07, 'hidden_layer_sizes': 82, 'learning_rate_init': 0.00546301016905825, 'tol': 0.00016667583915584017, 'validation_fraction': 0.1945260244214272}]
function_evaluation time 0.922634 value 3726.238213 suggestion {'alpha': 1.1645647017687961, 'batch_size': 139, 'beta_1': 0.7486998016452818, 'beta_2': 0.9902065016382664, 'epsilon': 7.344196077178596e-07, 'hidden_layer_sizes': 82, 'learning_rate_init': 0.00546301016905825, 'tol': 0.00016667583915584017, 'validation_fraction': 0.1945260244214272}
observation time 0.000006, current best 3204.059810 at iter 6
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 49, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.461145 iter 7 next_points [{'alpha': 0.0010851081293771062, 'batch_size': 118, 'beta_1': 0.9472668436936955, 'beta_2': 0.9157255555067895, 'epsilon': 9.522280553150698e-09, 'hidden_layer_sizes': 167, 'learning_rate_init': 0.00016433972641542542, 'tol': 0.0027562254377254706, 'validation_fraction': 0.25223858700080387}]
function_evaluation time 0.090608 value 29105.322421 suggestion {'alpha': 0.0010851081293771062, 'batch_size': 118, 'beta_1': 0.9472668436936955, 'beta_2': 0.9157255555067895, 'epsilon': 9.522280553150698e-09, 'hidden_layer_sizes': 167, 'learning_rate_init': 0.00016433972641542542, 'tol': 0.0027562254377254706, 'validation_fraction': 0.25223858700080387}
observation time 0.000006, current best 3204.059810 at iter 7
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 49, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.325311 iter 8 next_points [{'alpha': 0.00016975249962914972, 'batch_size': 184, 'beta_1': 0.9760509666471151, 'beta_2': 0.9475632993142001, 'epsilon': 1.9139287772146227e-07, 'hidden_layer_sizes': 105, 'learning_rate_init': 0.001201709160342984, 'tol': 0.03619215970130796, 'validation_fraction': 0.5265149269340205}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

function_evaluation time 0.056064 value 29008.119159 suggestion {'alpha': 0.00016975249962914972, 'batch_size': 184, 'beta_1': 0.9760509666471151, 'beta_2': 0.9475632993142001, 'epsilon': 1.9139287772146227e-07, 'hidden_layer_sizes': 105, 'learning_rate_init': 0.001201709160342984, 'tol': 0.03619215970130796, 'validation_fraction': 0.5265149269340205}
observation time 0.000005, current best 3204.059810 at iter 8
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 49, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.324973 iter 9 next_points [{'alpha': 0.5368640822949827, 'batch_size': 240, 'beta_1': 0.9697071278496382, 'beta_2': 0.9979449075815207, 'epsilon': 5.779234572196693e-09, 'hidden_layer_sizes': 139, 'learning_rate_init': 5.942828518113989e-05, 'tol': 2.296555820548193e-05, 'validation_fraction': 0.8713139272849658}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

function_evaluation time 0.713138 value 29077.322199 suggestion {'alpha': 0.5368640822949827, 'batch_size': 240, 'beta_1': 0.9697071278496382, 'beta_2': 0.9979449075815207, 'epsilon': 5.779234572196693e-09, 'hidden_layer_sizes': 139, 'learning_rate_init': 5.942828518113989e-05, 'tol': 2.296555820548193e-05, 'validation_fraction': 0.8713139272849658}
observation time 0.000006, current best 3204.059810 at iter 9
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 49, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.330542 iter 10 next_points [{'alpha': 0.015048767062750532, 'batch_size': 34, 'beta_1': 0.7367704806091967, 'beta_2': 0.9985463504384631, 'epsilon': 7.235939828218586e-09, 'hidden_layer_sizes': 172, 'learning_rate_init': 0.007415883800737713, 'tol': 0.013501170180929846, 'validation_fraction': 0.7603190950429273}]
function_evaluation time 0.417297 value 4114.526642 suggestion {'alpha': 0.015048767062750532, 'batch_size': 34, 'beta_1': 0.7367704806091967, 'beta_2': 0.9985463504384631, 'epsilon': 7.235939828218586e-09, 'hidden_layer_sizes': 172, 'learning_rate_init': 0.007415883800737713, 'tol': 0.013501170180929846, 'validation_fraction': 0.7603190950429273}
observation time 0.000006, current best 3204.059810 at iter 10
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 49, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.323778 iter 11 next_points [{'alpha': 0.047097292342261864, 'batch_size': 143, 'beta_1': 0.7060962601484314, 'beta_2': 0.999998467374711, 'epsilon': 2.3169629373347635e-08, 'hidden_layer_sizes': 128, 'learning_rate_init': 0.0011122882116348619, 'tol': 0.004083891021347884, 'validation_fraction': 0.5392462241942131}]
function_evaluation time 0.059898 value 29034.491117 suggestion {'alpha': 0.047097292342261864, 'batch_size': 143, 'beta_1': 0.7060962601484314, 'beta_2': 0.999998467374711, 'epsilon': 2.3169629373347635e-08, 'hidden_layer_sizes': 128, 'learning_rate_init': 0.0011122882116348619, 'tol': 0.004083891021347884, 'validation_fraction': 0.5392462241942131}
observation time 0.000006, current best 3204.059810 at iter 11
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 49, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.320794 iter 12 next_points [{'alpha': 3.8016507934212014e-05, 'batch_size': 134, 'beta_1': 0.9209530979863488, 'beta_2': 0.999967931453529, 'epsilon': 4.1388910309249716e-08, 'hidden_layer_sizes': 171, 'learning_rate_init': 0.0005659198761305832, 'tol': 0.02248980339164876, 'validation_fraction': 0.6236530671303081}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:605: UserWarning: Got `batch_size` less than 1 or larger than sample size. It is going to be clipped
  warnings.warn(

function_evaluation time 0.062497 value 29117.472968 suggestion {'alpha': 3.8016507934212014e-05, 'batch_size': 134, 'beta_1': 0.9209530979863488, 'beta_2': 0.999967931453529, 'epsilon': 4.1388910309249716e-08, 'hidden_layer_sizes': 171, 'learning_rate_init': 0.0005659198761305832, 'tol': 0.02248980339164876, 'validation_fraction': 0.6236530671303081}
observation time 0.000006, current best 3204.059810 at iter 12
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 49, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.323946 iter 13 next_points [{'alpha': 5.512587131461958e-05, 'batch_size': 11, 'beta_1': 0.7710212092048204, 'beta_2': 0.9996271357518005, 'epsilon': 2.7866845480962797e-07, 'hidden_layer_sizes': 152, 'learning_rate_init': 0.002541422505442943, 'tol': 0.01066505834105172, 'validation_fraction': 0.26716201348828406}]
function_evaluation time 0.706753 value 3841.662008 suggestion {'alpha': 5.512587131461958e-05, 'batch_size': 11, 'beta_1': 0.7710212092048204, 'beta_2': 0.9996271357518005, 'epsilon': 2.7866845480962797e-07, 'hidden_layer_sizes': 152, 'learning_rate_init': 0.002541422505442943, 'tol': 0.01066505834105172, 'validation_fraction': 0.26716201348828406}
observation time 0.000007, current best 3204.059810 at iter 13
Failure in optimizer suggest. Falling back to random search.
name 'activ' is not defined
Traceback (most recent call last):
  File "/home/ryedida/.local/lib/python3.9/site-packages/bayesmark/experiment.py", line 126, in run_study
    next_points = optimizer.suggest(n_suggestions)
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in suggest
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 109, in <listcomp>
    betas = [self._get_smoothness_reg(guess) for guess in guesses]
  File "/home/ryedida/bbo_challenge_starter_kit/example_submissions/smoothness/optimizer.py", line 49, in _get_smoothness_reg
    if activ > Kz:
NameError: name 'activ' is not defined
suggestion time taken 0.322503 iter 14 next_points [{'alpha': 0.00014793815258636812, 'batch_size': 100, 'beta_1': 0.849749410311943, 'beta_2': 0.999997791692446, 'epsilon': 1.3469803100504838e-09, 'hidden_layer_sizes': 185, 'learning_rate_init': 1.7730687256418497e-05, 'tol': 4.9343316813142315e-05, 'validation_fraction': 0.10086066528351045}]
/home/ryedida/.local/lib/python3.9/site-packages/sklearn/neural_network/_multilayer_perceptron.py:686: ConvergenceWarning: Stochastic Optimizer: Maximum iterations (200) reached and the optimization hasn't converged yet.
  warnings.warn(

function_evaluation time 0.677633 value 29114.454931 suggestion {'alpha': 0.00014793815258636812, 'batch_size': 100, 'beta_1': 0.849749410311943, 'beta_2': 0.999997791692446, 'epsilon': 1.3469803100504838e-09, 'hidden_layer_sizes': 185, 'learning_rate_init': 1.7730687256418497e-05, 'tol': 4.9343316813142315e-05, 'validation_fraction': 0.10086066528351045}
observation time 0.000007, current best 3204.059810 at iter 14
saving meta data: {'args': {'--uuid': '4dd5610812c15cc9b23b221141bb85b0', '-db-root': '/home/ryedida/bbo_challenge_starter_kit/output', '--opt-root': '/home/ryedida/bbo_challenge_starter_kit/example_submissions', '--data-root': None, '--db': 'run_20230427_004608', '--opt': 'smoothness', '--data': 'diabetes', '--classifier': 'MLP-adam', '--metric': 'mse', '--calls': 15, '--suggestions': 1, '--jobs-file': None, '--verbose': False, 'dry_run': False, 'rev': 'a376313', 'opt_rev': 'x.x.x'}, 'signature': ([{'alpha': 0.019628224813442792, 'batch_size': 182, 'beta_1': 0.9410202200271762, 'beta_2': 0.9998021557676793, 'epsilon': 1.8662266976518e-08, 'hidden_layer_sizes': 147, 'learning_rate_init': 0.0005627932047415167, 'tol': 0.03690557729213761, 'validation_fraction': 0.8846827852548593}, {'alpha': 0.0019982467392329444, 'batch_size': 200, 'beta_1': 0.919111482530466, 'beta_2': 0.9998488260436156, 'epsilon': 5.981221901152555e-07, 'hidden_layer_sizes': 61, 'learning_rate_init': 2.2310905607443014e-05, 'tol': 1.2046852412030316e-05, 'validation_fraction': 0.8117896445826539}, {'alpha': 0.46659545670218433, 'batch_size': 219, 'beta_1': 0.9889789783750891, 'beta_2': 0.9999896868093284, 'epsilon': 2.4234724484675948e-08, 'hidden_layer_sizes': 167, 'learning_rate_init': 2.972334644335654e-05, 'tol': 0.0036281404040243792, 'validation_fraction': 0.17260651658522078}, {'alpha': 4.656005689076002, 'batch_size': 135, 'beta_1': 0.870503881627747, 'beta_2': 0.9948873266941017, 'epsilon': 2.1023308743480125e-07, 'hidden_layer_sizes': 118, 'learning_rate_init': 0.0018781738757161913, 'tol': 1.1889379831773004e-05, 'validation_fraction': 0.6264327093752792}, {'alpha': 0.04705159350400542, 'batch_size': 158, 'beta_1': 0.9870884262730957, 'beta_2': 0.9999596874382349, 'epsilon': 1.1981845126013875e-08, 'hidden_layer_sizes': 116, 'learning_rate_init': 0.00617340520407431, 'tol': 1.741413418158619e-05, 'validation_fraction': 0.6754299026638921}], [29083.993548572587, 29062.423277324073, 29075.78987471409, 24626.32866834337, 9584.226858493177])}
saving results
saving timing
saving suggest log
done
